apps/web/src/hooks/use-SearchParams.ts

```diff
@@ -0,0 +1,13 @@
+import { useMemo } from 'react';
```

Comment:
I think filename should be `useSearchParams`, camelcased like the other hook files we have.

Reply:
I did see we do like this for other hooks... But I agree with you 😄 
<img width="286" alt="Screenshot 2023-01-03 at 11 30 40" src="https://user-images.githubusercontent.com/2233092/210340141-47d26e2e-e9b3-4a19-abad-d9f2be85c4bd.png">

Reply:
Ah damn we need to define a standard. I just quickly checked other hooks. Either kebab-case or camelCase should be fine. You went totally diplomatic using both. 🙂

Reply:
Haha well we have bot as well since before 🤣 
<img width="304" alt="Screenshot 2023-01-03 at 11 51 18" src="https://user-images.githubusercontent.com/2233092/210343288-b4440a9e-dc9a-4a96-82d4-ec2f318d1da5.png">

Reply:
🙃 🔫

Reply:
I fixed these for my hooks now.